.travis.yml

```diff
@@ -44,6 +47,7 @@ install:
   - python -m pip install -U pip
   - pip install tensorflow==${TENSORFLOW} keras==${KERAS}
   - pip install torch==${PYTORCH} torchvision==${TORCHVISION}
```

Comment:
you can append `pytorch_lighning` to the line above, as you do in `Dockerfile.cpu`

Reply:
sure